### DISCUSSION TITLE
Can I use correction factors computed for a reference image and apply them to others?

### DISCUSSION THREAD

**Discussion by fredonehouse** [colour-science/colour-checker-detection#62] [General]
I have a series of images taken at the same time taken under the same conditions. One of them is the Color Checker. I am trying to determine the correction factors (swatches, REFERENCE_SWATCHES) with the help of colour_checker_detection and apply them to the rest of the images. Is it rigth to use the method colour.colour_correction(OTHER_IMAGES[i], swatches, REFERENCE_SWATCHES) to correct the other images with the previously calculated factors?
Is it additionally possible to use Nikon's NEF format directly instead of taking the detour via png files?

Thanks!

**Comment by KelSolaar** [comment]
Hello @fredonehouse,

Absolutely fine to use the same factors if the images have been processed the same way and are under the same illumination conditions.

In order to use Nikon's NEF, you will need something to read them, e.g. Rawpy.

Cheers,

Thomas
